Dockerfile

```diff
@@ -9,14 +9,18 @@ ENV GHPRBHOOK_USER=CHANGE_ME
 ENV GHPRBHOOK_PASS=CHANGE_ME
 ENV QPS_HOST=localhost
 ENV QPS_PIPELINE_GIT_URL=https://github.com/qaprosoft/qps-pipeline.git
-ENV QPS_PIPELINE_GIT_BRANCH=4.10.2
+ENV QPS_PIPELINE_GIT_BRANCH=4.10.3
 ENV JENKINS_OPTS="--prefix=/jenkins --httpPort=8080"
-#ENV JENKINS_OPTS="--prefix=/jenkins --httpPort=-1 --httpsPort=8083 --httpsKeyStore=/var/jenkins_home/keystore.jks --httpsKeyStorePassword=password"
 ENV AWS_KEY=CHANGE_ME
 ENV AWS_SECRET=CHANGE_ME
 ENV QPS_PIPELINE_LOG_LEVEL=INFO
 ENV LANG=en_US.UTF-8
 ENV LANGUAGE=en_US.UTF-8
+ENV SONAR_NAME=sonar-ci
+ENV SONAR_URL=http://change_me.com
```

Comment:
default value should be: http://sonarqube:9000/sonarqube
in this case as you remember it will work in scope of infra automatically